convert2rhel/utils.py

```diff
@@ -538,8 +538,9 @@ def require_single_user_mode():
     """
     loggerinst = logging.getLogger(__name__)
 
-    output, ret_code = run_cmd_in_pty("runlevel", print_cmd=False)
-    runlevel = output.replace("\r\n", "")[-1]
+    runlevel_path = pexpect.which("runlevel")
+    output, ret_code = run_subprocess(cmd=runlevel_path, print_cmd=False)
+    runlevel = output.replace("\r\n", "").replace("\n", "")[-1]
```

Comment:
```suggestion
    runlevel = output.split()[-1]
```
The output is `N <number>`, e.g. `N 1`. You can use `split()` to split it to `['N', '1']`.

Reply:
maybe also int(....) to have a valid int